.run/Rest API - JDBC - license.run.xml

```diff
@@ -0,0 +1,21 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Rest API - JDBC - license" type="Application" factoryName="Application" folderName="Rest API">
+    <envs>
+      <env name="GRAVITEE_MANAGEMENT_JDBC_DATABASE" value="gravitee" />
+      <env name="GRAVITEE_MANAGEMENT_JDBC_DRIVER" value="postgresql" />
+      <env name="GRAVITEE_MANAGEMENT_JDBC_HOST" value="localhost" />
+      <env name="GRAVITEE_MANAGEMENT_JDBC_PASSWORD" value="admin" />
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential 2   
**Severity:** HIGH   
**Match:**       <env name="GRAVITEE_MANAGEMENT_JDBC_PASSWORD" value="*****" />  
[This comment was created by Aqua Pipeline]

Reply:
Removed